js/main.js

```diff
@@ -176,7 +197,83 @@ var generatePins = function (array) {
   map.querySelector('.map__pins').appendChild(fragment);
 };
 
-getAds(ads, TOTAL_ADS);
-generatePins(ads);
+// добавляет или удаляет атрибуты disabled, в зависимости от флага isActive
+var changeFieldsetsState = function (array, isActive) {
+  if (isActive) {
+    for (var i = 0; i < array.length; i++) {
+      array[i].removeAttribute('disabled', 'disabled');
+    }
+  } else {
+    for (var j = 0; j < array.length; j++) {
+      array[j].setAttribute('disabled', 'disabled');
+    }
+  }
+};
+
+// заполняет поле с адресом
+var addAddress = function (isActive) {
+  if (isActive) {
+    addressField.value = (Math.round(pinMainSelector.offsetLeft + pin.getPointer(pin.mainWidth))) + ', ' + (Math.round(pinMainSelector.offsetTop + pin.mapHeight)); // координаты острого конца метки
```

Comment:
```suggestion
    addressField.value = (Math.round(pinMainSelector.offsetLeft + pin.getPointer(pin.mainWidth))) + ', ' + (Math.round(pinMainSelector.offsetTop + **pin.mapHeight**)); // координаты острого конца метки
```
А я ещё читая код подумала, почему у тебя в словаре pin есть `xMax: mapWidth` ширина карты, а нет высоты карты, `var mapHeight = map.offsetHeight;` 94 строка. 

Но тут ты может имела в виду `mainHeight`

Reply:
Это у тебя адрес ушел за экран и его не видно. Лайфхак: по плюсам можно переносить на другую строку (складывать). Надо чтобы она начиналась с плюса:
```
  if (isActive) {
    // координаты острого конца метки
    addressField.value = (Math.round(pinMainSelector.offsetLeft + pin.getPointer(pin.mainWidth)))
      + ', '
      + (Math.round(pinMainSelector.offsetTop + pin.mapHeight));
  } else {
    // если страница не активна, координаты метки - центр карты
    addressField.value = (Math.round(mapWidth / 2))
      + ', '
      + (Math.round(mapHeight / 2));
  }
```

Reply:
Коменты обычно пишутся перед кодом